### PR TITLE
Fix to capture codeowner plus ability to run on existing PRs

### PR DESCRIPTION
**Pull request by desi**
## Explanation

Right now for PRs where the WF team are codeowners but our label isn't there or we aren't specifically added as requested reviewers, they aren't getting added to our repo for review. This PR is intended to fix that.

## Changelog

n/a

## Checklist
n/a

### CODE REVIEW

File: .github/workflows/add-prs-to-project.yml [hunk 1]
```diff
@@ -14,11 +19,25 @@ jobs:
       TEAM_LABEL: 'team-wallet-framework'
 
     steps:
+      - name: Get existing PRs
+        if: github.event.inputs.scan_existing_prs == 'true'
+        id: get-prs
```
Comment (mcmire) [inline .github/workflows/add-prs-to-project.yml:24]: Where is `get-prs` being used? It's not referenced in this workflow so I'm a little confused 🤔

Comment (desi) [reply .github/workflows/add-prs-to-project.yml:24]: The inputs area above in `workflow_dispatch` should allow me to run this manually in the github actions view and select "true" the input field. That will trigger `get-prs` to run. 

https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/

I have not tested this on a fork but given it should only impact a manual run and shouldn't cause any harm I was hoping to be able to test it once merged.

Comment (mcmire) [reply .github/workflows/add-prs-to-project.yml:24]: Oh sorry — I meant that I see how this step could get run, and I see that this step reads a list of pull requests, but I don't see where those pull requests are getting used. Does the `add-to-project` implicitly use the results of this step somehow?

**Review by mcmire — APPROVED** [review]
Looks good!

**Pull request merged**